### PR TITLE
bpo-32069: Drop legacy SSL transport

### PR DESCRIPTION
**Pull request by asvetlov**
Legacy SSL transports was not used starting from Python 3.5, let's drop never executed code.

https://bugs.python.org/issue32069

### CODE REVIEW

File: Lib/test/test_asyncio/test_selector_events.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
 
 import asyncio
 from asyncio import selectors
+from asyncio import sslproto
```
Comment (1st1) [inline Lib/test/test_asyncio/test_selector_events.py]: Why this new import?

**Review by 1st1 — APPROVED** [review]

File: Lib/test/test_asyncio/test_selector_events.py (hunk 1 above)
Comment (asvetlov) [reply Lib/test/test_asyncio/test_selector_events.py]: Was my mistake, dropped.

**Comment by 1st1** [comment]
Hey, this must not be merged to 3.6

**Comment by asvetlov** [comment]
Why?
The code in 3.6 will still keep working, nobody will see the change!
If you strongly disagree I'll make a revert commit but please tell me why?
All changes are for dropping 3.4 only features.

**Comment by 1st1** [comment]
3.6 is in a bug fix mode. Removing **any** APIs does not match that, unless it’s a critical security issue.

Maybe someone is importing/using these APIs manually. Breaking their code (however wrong it is) in a bugfix release is not an option.

**Comment by asvetlov** [comment]
I've  got your point.
Closing #4453

**Comment by asvetlov** [comment]
Thanks for pointing on the problem.

**Comment by 1st1** [comment]
Thanks!

**Pull request merged**